src/cephadm/tox.ini

```diff
@@ -59,7 +59,7 @@ commands =
 basepython = python3
 allowlist_externals = bash
 deps =
-    flake8
+    flake8 == 5.0.4
```

Comment:
Out of curiosity, is there any something special about this version or because it's just the latest?

Reply:
explained in the commit message.